Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.9-buster as base
+# Perform common operations, dependency installation etc...
+RUN apt-get update && apt-get install -y \
+    curl \
+    git
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+# Adding poetry to PATH
+ENV PATH="/root/.poetry/bin:$PATH"
+
+RUN mkdir /app/
+COPY . /app/
```

Comment:
You are currently including your .env file in the image. As the instructions explained, you should make sure not to include secrets in the image. You can do this by adding `.env` to your `.dockerignore` file.

You instead pass the env vars to the container as part of your `docker run` command, or via docker-compose.

Your dockerignore should also include things the image doesn't need, like `.git` `Dockerfile` `README.md` `.vagrant`, `Vagrantfile` etc

That will mean you avoid some unnecessary rebuilds and the image is smaller.

Reply:
Updated